vllm/triton_utils/importing.py

```diff
@@ -1,11 +1,16 @@
 from importlib.util import find_spec
 
 from vllm.logger import init_logger
+from vllm.platforms import current_platform
 
 logger = init_logger(__name__)
 
+if not current_platform.is_xpu():
+    logger.info("!!! not xpu")
+
```

Comment:
remove debug log?

Reply:
This is for debugging the CI failures. Now removed.